cpp/oneapi/dal/algo/jaccard/backend/cpu/vertex_similarity_default_kernel_avx2.hpp

```diff
@@ -265,10 +265,12 @@ vertex_similarity_result call_jaccard_default_kernel_avx2(
     const descriptor_base &desc,
     vertex_similarity_input<undirected_adjacency_array_graph<>> &input) {
     const auto &my_graph = input.get_graph();
-    const auto &g = oneapi::dal::preview::detail::get_impl(my_graph);
-    auto g_edge_offsets = g->_edge_offsets.data();
-    auto g_vertex_neighbors = g->_vertex_neighbors.data();
-    auto g_degrees = g->_degrees.data();
+    using graph_type = undirected_adjacency_array_graph<>;
+    const auto &g =
+        oneapi::dal::detail::get_impl<const typename graph_traits<graph_type>::impl_type>(my_graph);
```

Comment:
Please synchronise your branch with master, there are the new changes that simplifies `get_impl`

```suggestion
        dal::detail::get_impl(my_graph);
```